hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -2657,6 +2657,17 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.datanode.round-robin-volume-choosing-policy.additional-available-space</name>
+  <value>0</value>
```

Comment:
Do you have a good default number in mind? IMO, if this configuration potentially saves a big problem (disk full) we should consider picking a reasonably good default number if it's not good big.

Reply:
Yes, Good suggestion. I think the reasonably good default number I want to set is 10 GB , which is the same as the default value of dfs.datanode.available-space-volume-choosing-policy.balanced-space-threshold. since AvailableSpaceVolumeChoosingPolicy uses this default value as balanced disk maximum difference. I think this value can safeguard the disk space and also not waste much disk storage.

Reply:
So if we pick 10 GB or another larger value, it may cause some issues for some overloaded DNs when applying this patch, right?

Reply:
No, It won't have issue. 10 GB is good for practical usage. But I am afraid It may have some issues if people test Hadoop in local mode with less than 10GB available space on the local disks without changing this configuration value. Either 10 GB or 4 MB works for me. thanks

Reply:
4MB looks too small here, given that DataXceiver threads count is configurable. How about 1GB? @zhihaixu2012, @jojochuang

Reply:
@ywskycn thanks for the review! Yes, Good Suggestion, I think 1GB is a good default number in this case.

Reply:
I just updated the patch with the default value 1GB.

Reply:
ok 1GB sounds fine to me.